platform/linux-generic/include/odp_pktio_ops_subsystem.h

```diff
@@ -88,10 +89,40 @@ typedef ODP_MODULE_CLASS(pktio_ops) {
 /* Maximum size of pktio specific ops data.*/
 #define ODP_PKTIO_ODPS_DATA_MAX_SIZE 80000
 
+/* Pktio ops data pool */
+#define ODP_PKTIO_OPS_DATA_POOL_SIZE 160000
+#define ODP_PKTIO_OPS_DATA_POOL_NAME "ODP_PKTIO_OPS_DATA"
+
 /* Extract pktio ops data from pktio entry structure */
 #define odp_ops_data(_p, _mod) \
```

Comment:
Look good no more comments from me to this commit after Honnappa and Josep's, this is a step forward for the pktio ops data

This commit also reveals how complex in ODP to allocate an arbitrary sized piece of memory, need to prepare a pool (and guess the largest usage), lookup this pool in every allocation/free by name, and do the allocation/free after then.

Reply:
In future, since each pktio_ops module will not expose their private data type, this macro can be changed to
>`#define odp_ops_data(_entry, _pdata) \
        pdata = (typeof(_pdata))(_entry->s.ops_data)`

So the odp_pktio_ops_subsystem.h header won't need to know all pktio_ops' private data structure declaration. Can be considered next time.